hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTaskWithLegacy.java

```diff
@@ -326,8 +326,11 @@ private void setKeyParentID(OmKeyInfo keyInfo) throws IOException {
       if (parentKeyInfo != null) {
         keyInfo.setParentObjectID(parentKeyInfo.getObjectID());
       } else {
-        throw new IOException("ParentKeyInfo for " +
-            "NSSummaryTaskWithLegacy is null");
+        LOG.error(
```

Comment:
Pls change it to warn level. This may not be error as Recon OM DB can go out of sync with OM DB, so we need to first make sure with elaborative warn log. Ideally parent object Id for a key should not be null, so we need to find if this is the same case with OM DB as well.

Reply:
Done!